src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $gap;
+  padding: $padding;
+  width: calendarWidth(
+    $day-size,
+    $colums-count,
+    $gap,
+    $padding
+  );
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    line-height: 100px;
+    text-align: center;
+    font-size: 30px;
+    border: 1px solid $color-black;
+    background-color: $color-white-grey;
+    transition: 0.5s;
```

Comment:
Add transition properties.
Add duration to variables.

Reply:
fixed